src/patternfly/components/Label/examples/Label.md

```diff
@@ -596,24 +603,36 @@ This style of label is used to indicate overflow within a label group.
 {{#> label label--id="compact-editable-label-active" label--modifier="pf-m-compact pf-m-blue" label--IsEditable="true" label--IsEditableActive="true"}}Compact editable active{{/label}}
 ```
 
+### Add label
+
+This style of label is used to add new labels to a label group.
+
+```hbs isBeta
+{{#> label label--id="add" label--IsAdd="true"}}
```

Comment:
Just a nit but I don't think this ID is necessary. Doesn't look necessary on a bunch of other labels, either, though we don't have to update those here.

```suggestion
{{#> label label--IsAdd="true"}}
```